packages/core/cli.js

```diff
@@ -65,48 +69,46 @@ command.run(inputArguments, options, function (err) {
       });
       command.displayGeneralHelp();
     } else {
-      if (err instanceof TruffleError) {
+      if (error instanceof TruffleError) {
```

Comment:
This can become an `else if ...`

Reply:
How about if this became a big `switch` statement? That's really what is happening here, ya?